src/pages/about-us/team/index.tsx

```diff
@@ -1,13 +1,45 @@
 // path: about-us/team/leadership
 
 import { Typography } from '@mui/material';
+import { GetServerSideProps } from 'next';
 
-const TeamLeadershipPage = () => {
+// import { TeamCoreData, TeamContactData, TeamMembersData } from '@utils/types';
+import { TeamApiResponse } from '@utils/types';
+import { fetchData } from 'lib/api';
+
+interface TeamPageProps {
+  team: TeamApiResponse;
+}
+
+const TeamLeadershipPage = ({ team }: TeamPageProps) => {
+  const { membersByType } = team;
   return (
     <div>
+      {membersByType.directors.map((director) => (
+        <span key={director.fullName}>{director.fullName}</span> //Example how to use the data
```

Comment:
I think you don't need add comment here, but you can describe in the PR this is an example how to use the API.